app/src/main/java/com/padc/nyi/moneysaver123/activities/HomeActivity.java

```diff
@@ -51,4 +92,65 @@ public boolean onOptionsItemSelected(MenuItem item) {
 
         return super.onOptionsItemSelected(item);
     }
+
+    @SuppressWarnings("StatementWithEmptyBody")
+    @Override
+    public boolean onNavigationItemSelected(MenuItem item) {
+        // Handle navigation view item clicks here.
+
+        drawer.closeDrawer(GravityCompat.START);
+
+        switch (item.getItemId()) {
+            case R.id.nav_expense:
+                navigateToExpense();
+                return true;
+            case R.id.nav_income:
+                navigateToIncome();
+                return true;
+            case R.id.nav_expense_category:
+                navigateToExpenseCategory();
+                return true;
+            case R.id.nav_bill:
+                navigateToBill();
+                return true;
+        }
+        return false;
+    }
+
+    //call fragment
+    private void navigateToExpense(){
+
+        ExpenseFragment expenseFragment = new ExpenseFragment();
+        getSupportFragmentManager()
+                .beginTransaction()
+                .replace(R.id.fl_container, expenseFragment)
+                .commit();
+    }
+
+    private void navigateToIncome(){
+
+        IncomeFragment incomeFragment = new IncomeFragment();
+        getSupportFragmentManager()
+                .beginTransaction()
+                .replace(R.id.fl_container, incomeFragment)
+                .commit();
+    }
+
+    private void navigateToExpenseCategory(){
+
+        ExpenseFragment expenseFragment = new ExpenseFragment();
```

Comment:
Need to use Static Factory Pattern in creating new fragment. For all IncomeFragment, ExpenseFragment & others.